### PR TITLE
Home 8 change detection

### PR DESCRIPTION
**Pull request by YKotov**

### CODE REVIEW

File: pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<article class="new-course-form">
+  <label for="title">Title
+    <input type="text" name="title">
+  </label>
+
+  <label for="desc">Description
```
Comment (torin4fan) [inline pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.html]: if you add (<label for="title") you should also add id attribute for some field.

File: pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  public duration: number;
+  @Output() closeForm: any = new EventEmitter();
+
+  constructor() {
+  }
+
+  ngOnInit() {
+  }
+
+  closeFormEv(e) {
```
Comment (torin4fan) [inline pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.ts]: put types for output and for attribute for all methods.

File: pet-app/src/app/pages/courses/actions/add-more/add-more.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import {AuthorizationService} from '../../../login/authorization.service';
+
+@Component({
+  selector: 'app-add-more',
+  templateUrl: './add-more.component.html',
+  styleUrls: ['./add-more.component.css']
+})
+export class AddMoreComponent implements OnInit {
+
+  public formAppearCl: string;
```
Comment (torin4fan) [inline pet-app/src/app/pages/courses/actions/add-more/add-more.component.ts]: make this property as boolean

File: pet-app/src/app/pages/courses/actions/add-more/add-more.component.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<div class="add-more">
+  <button (click)="addMoreForm()">Add more!</button>
+  <app-new-course class="add-more-el" ngClass={{formAppearCl}} (closeForm)="closeAddMoreForm()"></app-new-course>
```
Comment (torin4fan) [inline pet-app/src/app/pages/courses/actions/add-more/add-more.component.html]: [ngClass] = {'show-add-form' : formAppearCl}

File: pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.html (hunk 1 above)
Comment (YKotov) [reply pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.html]: miss understand

File: pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.ts (hunk 2 above)
Comment (YKotov) [reply pet-app/src/app/pages/courses/actions/add-more/new-course/new-course.component.ts]: do not understand ((

File: pet-app/src/app/pages/courses/actions/add-more/add-more.component.html (hunk 4 above)
Comment (YKotov) [reply pet-app/src/app/pages/courses/actions/add-more/add-more.component.html]: done

File: pet-app/src/app/pages/courses/actions/add-more/add-more.component.ts (hunk 3 above)
Comment (YKotov) [reply pet-app/src/app/pages/courses/actions/add-more/add-more.component.ts]: done

**Pull request merged**